### PR TITLE
Add FLUXNET data for ClimaLand/experiments

### PR DESCRIPTION
**Pull request by Espeer5**
This PR adds an artifact for flux tower data used in ClimaLand experiments. This is part of [an effort to remove the dependency on ArtifactWrapper from ClimaLand.jl](https://github.com/CliMA/ClimaLand.jl/issues/580).

Checklist:
- [x] I created a new folder `$artifact_name`
  - [x] I added a `README.md` in that that folder that
    - [x] describes the data and processing done to it
    - [x] lists the sources of the raw data
    - [x] lists the required citation, licenses
  - [x] If applicable (e.g., for Creative Commons), I added a `LICENSE` file
  - [x] I added the scripts that retrieve, process, and produce the artifact
  - [x] I added the environment used for such scripts (typically, `Project.toml`
        and `Manifest.toml`)
  - [x] I added the `OutputArtifacts.toml` file containing the information
        needed for package developers to add `$artifact_name` to their package
- [x] I uploaded the artifact folder to the Caltech cluster (in
      `/groups/esm/ClimaArtifacts/artifacts/$artifact_name`)
- [x] I added the relevant code to the `Overides.toml` on the Caltech Cluster
      (in `/groups/esm/ClimaArtifacts/artifacts/Overrides.toml`)
- [x] I added a link to the main `README.md` to point to the new artifact

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ S. Gupta et al 2022, 2024](https://github.com/CliMA/ClimaArtifacts/tree/main/soi
 - [Bonan 2019 Richards equation data](https:////github.com/CliMA/ClimaArtifacts/tree/main/bonan_richards_eqn)
 - [TwoStream model implementation test data](https:////github.com/CliMA/ClimaArtifacts/tree/main/twostr_test)
 - [Berkeley Earth Global Gridded Temperature Data](https:////github.com/CliMA/ClimaArtifacts/tree/main/surface_temperatures)
+- [Select FLUXNET tower data](https:////github.com/CliMA/ClimaArtifacts/tree/main/fluxnet_sites)
```
Comment (Sbozzolo) [inline README.md:55]: Should this be?
```suggestion
- [Selected FLUXNET tower data](https:////github.com/CliMA/ClimaArtifacts/tree/main/fluxnet_sites)
```

File: fluxnet_sites/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+To download the data, visit the fluxnet data download page here: https://ameriflux.lbl.gov/data/download-data/. To download, you must first make an account, and then fill out a brief form indicating your intended use of the data. Once you have access, download the data for the following sites:
+ - US-Ha1 
+ - US-MOz
+ - US-NR1
+ - US-Var
+We use the FULLSET hourly data product which should be named:
+AMF_US-***_FLUXNET_FULLSET_HR_YYYY-YYYY_3-5.csv
+Place this file for each site where the site ID correcsponds to the '***' in the file name in the `fluxnet_site_data` directory. Then run this script to create the artifact. In creating the artifacts, we trim down the data to a certain span of years to make the data more manageable.
+
+License: CC BY 4.0
```
Comment (Sbozzolo) [inline fluxnet_sites/README.md:15]: Is the license CC BY 4.0, or the PDF you attach?

**Review by Sbozzolo — APPROVED** [review]

File: fluxnet_sites/README.md (hunk 2 above)
Comment (Espeer5) [reply fluxnet_sites/README.md:15]: If you look at the PDF, it's just a concise statement of how CC BY 4.0 applies to the Ameriflux data. They distribute that PDF along with their data so I attached it but it is really just a statement explaining that Ameriflux is under CC BY 4.0 along with a link to where to find proper citations for each site.

Comment (Sbozzolo) [reply fluxnet_sites/README.md:15]: Sounds good!

**Pull request merged**